matryoshka_tester/data.py

```diff
@@ -12,13 +12,13 @@
     "16": "registry.opensuse.org/home/fcrozat/matryoshka/containerfile/bci/openjdk-devel:16",
     "15": "registry.opensuse.org/home/fcrozat/matryoshka/containerfile/bci/openjdk-devel:15",
     "14": "registry.opensuse.org/home/fcrozat/matryoshka/containerfile/bci/openjdk-devel:14",
-    "11": "registry.opensuse.org/home/fcrozat/matryoshka/containerfile/bci/openjdk-devel:11",
```

Comment:
Why did you only change one of them?

Reply:
Because this is work in progress and I only have 1 of them at the moment on the new repository!